### PR TITLE
Remove from README info duplicated in docs

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Review by sfc-gh-jsikorski — APPROVED** [review]

**Review by sfc-gh-astus — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,141 +70,6 @@ snow --version
 
 You should now be able to run `snow` and get the CLI message.
```
Comment (sfc-gh-mheavin) [inline README.md:72]: I suggest deleting starting with line 11 (Overview), line 19 (Benefits) or line 30 (Limitations).

Or, what do you think of this:

Note text (remove For complete installation...)

Overview

Benefits

For complete installation...

Get involved

**Review by sfc-gh-mheavin — COMMENTED** [review]
Maybe we should trim more

File: README.md (hunk 1 above)
Comment (sfc-gh-turbaszek) [reply README.md:47]: I removed more but left installation instruction so people can easily get the tool

**Review by sfc-gh-pjob — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,49 +8,24 @@ Snowflake sales representative.
 For complete installation and usage instructions, refer to the
 [Snowflake CLI Guide](https://docs.snowflake.com/LIMITEDACCESS/snowcli/snowcli-guide).
```
Comment (sfc-gh-anugupta) [inline README.md:9]: Can we make it bold ?

Comment (sfc-gh-turbaszek) [reply README.md:9]: Sure!

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**